Site/bc_weather_station_dashboard/website/views.py

```diff
@@ -81,3 +88,59 @@ def submit_feedback(request):
             return redirect("home")
 
     return redirect("home")
+
+def station_data(request):
+    # Get the selected date from the query
+    selected_date = request.GET.get('datetime', None)
+    # Check if date is undefined
+    if selected_date is "undefined":
```

Comment:
Needs to be
if selected_date == "undefined":

Reply:
Done.